data/vendor-customer-support.ini

```diff
@@ -0,0 +1,8 @@
+[Customer Support]
```

Comment:
Hmm would it be possible to use the regular translation infrastructure for this instead?